python/tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = py27
+skipdist = true
+
+[testenv]
+deps =
+        -rrequirements.txt
```

Comment:
just wanted to make sure this is intended and not a typo?

Reply:
Oh yes, that could look like a typo! but it's not :)

When checking the options for pip install we get:
```
$ pip install --help
[...]
  -r, --requirement <file>    Install from the given requirements file. This option can be used multiple times.
```

And the space between -r and requirements.txt is not required.